### ISSUE TITLE
1.1.1 Non-text Content

### ISSUE THREAD

**Issue by WBroadwell** [LabConnect-RCOS/LabConnect-Frontend#49]
- Images, image buttons, and image map hot spots have appropriate, equivalent [alternative text].
- Images that do not convey content, are decorative, or contain content that is already conveyed in text are given empty alternative text (alt="") or implemented as CSS backgrounds. All linked images have descriptive alternative text.
- Equivalent alternatives to complex images are provided in context or on a separate linked page.
- Form buttons have a descriptive value.
- Inputs have associated [accessible names](https://webaim.org/articles/label-name/).
- Embedded multimedia is identified via accessible text.
- Frames and iframes are appropriately [titled](https://webaim.org/techniques/frames/).

**Comment by WBroadwell** [comment]
added alt text to all images, accessible names should be good since out Input component uses label field
